src/agent/services/lazer_exporter.rs

```diff
@@ -156,10 +236,26 @@ async fn fetch_symbols(history_url: &Url) -> Result<Vec<SymbolResponse>> {
 
 #[instrument(skip(config, state))]
 pub fn lazer_exporter(config: Config, state: Arc<state::State>) -> Vec<JoinHandle<()>> {
-    let handles = vec![tokio::spawn(lazer_exporter::lazer_exporter(
+    let mut handles = vec![];
+    let mut relayer_senders = vec![];
+
+    for url in config.relayer_urls.iter() {
+        let (sender, receiver) = mpsc::channel(RELAYER_CHANNEL_CAPACITY);
```

Comment:
you can use a broadcast::channel here instead of managing multiple single consumer channels